modules/ROOT/pages/ip-whitelist.adoc

```diff
@@ -7,68 +7,66 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Whitelist
->s|Summary      | Allows request only from a list or range of IP addresses
+>s|Summary      | Allows access to request from a list or range of specified IP addresses
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
 
-The IP Whitelist policy allows a list or range of IP addresses to access and interact with a configured API endpoint.
+The IP Whitelist policy allows a list or range of IP addresses to access a configured API endpoint.
 
-==  Whitelisting IP addresses to access your API endpoint
+The IP Whitelist policy allows access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
 
-You can set up access in the following ways:
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule)) or MEL expression for Mule 3.
 
-* Access based on the IP address of the request
-* Access based on the origin IP address of the x-Forwarded-For header
 
-== Whitelisting Based on the IP address of the Request
+[[specifying_ip_address]]
+== Specifying IP Addresses to Whitelist in Your Policy 
```

Comment:
BTW, I have MISLED you about topic set titles. 
I am told that we do not consider an .adoc a set of topics but rather one topic with subsections.
SO...please use imperative rather than gerund (-ing):
```suggestion
== Specify IP Addresses to Whitelist in Your Policy 
```

Reply:
This is GLOBAL going forward. Do not worry about past published content, but please use imperative titles for all tasks (ever super-tasks) going forward. I'm sorry!

Reply:
Yes, done! Thanks!